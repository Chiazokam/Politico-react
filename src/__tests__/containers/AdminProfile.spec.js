import React from 'react';
import { shallow } from 'enzyme';
import AdminProfile from '../../containers/admin-profile/Layout';

describe('<Admin Profile />', () => {
  it('renders correctly', () => {
    const props = {
    };
    const tree = shallow(<AdminProfile {...props} />);
    expect(tree).toMatchSnapshot(); 
  });
});
