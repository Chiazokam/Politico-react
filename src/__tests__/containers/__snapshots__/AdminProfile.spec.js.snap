// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Admin Profile /> renders correctly 1`] = `ShallowWrapper {}`;
